### ISSUE TITLE
Further improve accessibility

### ISSUE THREAD

**Issue by jaredgalanis** [eclipse-pass/main#745]
The prior accessibility tickets #717 and #716 focused on resolving explicit violations of accessibility standards.

This ticket seeks to ensure that our code follows accessibility best practices where possible.

The Axe Dev Accessibility Chrome Extension can support this effort as it has an additional check for these practices.
